Lib/fontTools/feaLib/lexer.py

```diff
@@ -193,14 +195,14 @@ def __init__(self, featurefile):
     def __iter__(self):
         return self
 
-    def next(self):  # Python 2
-        return self.__next__()
+    def next(self, comments = False):  # Python 2
```

Comment:
Style nitpick from [PEP8](https://www.python.org/dev/peps/pep-0008/#other-recommendations): “Don’t use spaces around the = sign when used to indicate a keyword argument or a default parameter value”.